main.py

```diff
@@ -79,6 +83,8 @@ def main():
     speaker_audio_recorder.record_into_queue(audio_queue)
 
     model = TranscriberModels.get_model('--api' in sys.argv)
+    combobox.configure(command=model.change_lang)
+    
```

Comment:
Small tip: is this extra black line at 87 needed?

Reply:
No it isn't. Fixed it thanks!